modules/backend/widgets/form/assets/js/october.form.js

```diff
@@ -39,9 +39,15 @@
 
         this.$el.on('oc.triggerOn.afterUpdate', this.proxy(this.toggleEmptyTabs))
         this.$el.one('dispose-control', this.proxy(this.dispose))
+        this.$form.closest('.modal').on('hidden.oc.popup', this.proxy(this.dispose))
     }
 
     FormWidget.prototype.dispose = function() {
+        this.unbindDependants();
+        this.unbindCheckboxList();
+        this.unbindLazyTabs();
+        this.unbindCollapsibleSections();
```

Comment:
@msimkunas We don't use semicolons in our JS code at the moment (even though I'd love to). Let's remove them for consistency.
```suggestion
        this.unbindDependants()
        this.unbindCheckboxList()
        this.unbindLazyTabs()
        this.unbindCollapsibleSections()
```